app/modules/import-export-customization/assets/js/import/components/drop-zone.js

```diff
@@ -0,0 +1,303 @@
+		const validExtensions = [];
+
+		filetypes.forEach( ( mimeType ) => {
+			if ( mimeToExtMap[ mimeType ] ) {
+				validExtensions.push( ...mimeToExtMap[ mimeType ] );
+			}
+		} );
+
+		return validExtensions;
+	};
+
+	const isValidFileType = ( fileType, fileName = '' ) => {
+		if ( 0 === filetypes.length ) {
+			return true;
+		}
+
+		if ( filetypes.includes( fileType ) ) {
+			return true;
+		}
+
+		const extension = fileName.toLowerCase().split( '.' ).pop();
+		const validExtensions = getValidExtensions();
+
+		return validExtensions.includes( extension );
+	};
+
+	const handleDragEnter = ( e ) => {
+		e.preventDefault();
+		e.stopPropagation();
+
+		setDragCounter( ( prev ) => prev + 1 );
+
+		if ( e.dataTransfer.items && e.dataTransfer.items.length > 0 ) {
+			setIsDragOver( true );
+		}
+	};
+
+	const handleDragLeave = ( e ) => {
+		e.preventDefault();
+		e.stopPropagation();
+
+		setDragCounter( ( prev ) => prev - 1 );
+
+		if ( dragCounter <= 1 ) {
+			setIsDragOver( false );
+		}
+	};
+
+	const handleDragOver = ( e ) => {
+		e.preventDefault();
+		e.stopPropagation();
+	};
+
+	const handleDrop = ( e ) => {
+		e.preventDefault();
+		e.stopPropagation();
+
+		setIsDragOver( false );
+		setDragCounter( 0 );
+
+		if ( isLoading ) {
+			return;
+		}
+
+		const file = e.dataTransfer.files[ 0 ];
+
+		if ( file && isValidFileType( file.type, file.name ) ) {
+			onFileSelect( file, e, 'drop' );
+		} else {
+			onError( {
+				id: 'file_not_allowed',
+				message: __( 'This file type is not allowed', 'elementor' ),
+			} );
+		}
+	};
+
+	const handleFileInputChange = ( e ) => {
+		const file = e.target.files[ 0 ];
+		if ( ! file ) {
+			return;
+		}
+
+		if ( ! isValidFileType( file.type, file.name ) ) {
+			onError( {
+				id: 'file_not_allowed',
+				message: __( 'This file type is not allowed', 'elementor' ),
+			} );
+			return;
+		}
+
+		onFileSelect( file, e, 'browse' );
+
+		if ( onFileChoose ) {
+			onFileChoose( file );
+		}
+	};
+
+	const handleButtonClick = ( e ) => {
+		if ( onButtonClick ) {
+			onButtonClick( e );
+		}
+
+		if ( fileInputRef.current ) {
+			fileInputRef.current.click();
+		}
+	};
+
+	const getAcceptedFileTypes = () => {
+		const acceptTypes = [ ...filetypes ];
+		const validExtensions = getValidExtensions();
+
+		validExtensions.forEach( ( ext ) => {
+			acceptTypes.push( `.${ ext }` );
+		} );
+
+		return acceptTypes.join( ',' );
+	};
+
+	return (
+		<Box
+			className={ className }
+			onDragEnter={ handleDragEnter }
+			onDragLeave={ handleDragLeave }
+			onDragOver={ handleDragOver }
+			onDrop={ handleDrop }
+			data-testid="drop-zone"
+			{ ...props }
+		>
+			<Box
+				sx={ {
+					border: 2,
+					borderColor: isDragOver ? 'primary.main' : 'divider',
+					borderStyle: 'dashed',
+					backgroundColor: isDragOver ? 'action.hover' : 'background.default',
+					transition: 'all 0.2s ease-in-out',
+					cursor: isLoading ? 'not-allowed' : 'normal',
+					position: 'relative',
+					overflow: 'hidden',
```

Comment:
```suggestion
					overflow: 'hidden',
```
Are you sure that it's needed?